hack/make/validate-lint

```diff
@@ -39,6 +39,9 @@ packages=(
 	pkg/version
 	registry
 	utils
+	volume
+	volume/local
+	volume/volumedrivers
```

Comment:
I'm still learning go, so this is confusing to me. The directory path is `volume/drivers`, but the actual package name is `volumedrivers` which leads me to think it should be `volume/volumedrivers`.